### PR TITLE
Add support for auto-loading

### PR DESCRIPTION
**Pull request by chb0github**
#111 fix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Jackson datatype module for [Vavr](http://vavr.io/) library
 <dependency>
   <groupId>io.vavr</groupId>
   <artifactId>vavr-jackson</artifactId>
-  <version>0.9.0</version>
+  <version>1.0.1</version>
```
Comment (ruslansennov) [inline README.md]: please keep `0.9.0` version

File: README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Developer versions can be found [here](https://oss.sonatype.org/content/reposito
 <dependency>
   <groupId>io.vavr</groupId>
   <artifactId>vavr-jackson</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>1.0.1-SNAPSHOT</version>
```
Comment (ruslansennov) [inline README.md]: `1.0.0-SNAPSHOT`

Comment (chb0github) [inline README.md]: Current version in the POM is `1.0.0-SNAPSHOT` Don't you want this to reflect the most current version?

File: README.md (hunk 1 above)
Comment (chb0github) [inline README.md]: Well, sure. But shouldn't it match the version given in the POM in this project?

Comment (ruslansennov) [inline README.md]: In the "Usage" paragraph of `README` we use [latest release](https://oss.sonatype.org/content/repositories/releases/io/vavr/vavr-jackson/) version of `vavr-jackson`, in "developer versions" we use [latest snapshot](https://oss.sonatype.org/content/repositories/snapshots/io/vavr/vavr-jackson/) version. Now it is `1.0.0-SNAPSHOT`, see [pom.xml at master branch](https://github.com/vavr-io/vavr-jackson/blob/master/pom.xml#L12)

File: src/test/java/io/vavr/jackson/datatype/ServiceLoaderTest.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package io.vavr.jackson.datatype;
```
Comment (ruslansennov) [inline src/test/java/io/vavr/jackson/datatype/ServiceLoaderTest.java]: Also, please add copyright header

Comment (ruslansennov) [inline src/test/java/io/vavr/jackson/datatype/ServiceLoaderTest.java]: oh... sorry, this is a test, it is unnecessary

File: README.md (hunk 1 above)
Comment (chb0github) [inline README.md]: Ok.

**Comment by chb0github** [comment]
I have done as requested. When do you think it might be released? No hurry. Just curious

**Comment by ruslansennov** [comment]
Thank you. 

Versions of `vavr-jackson` follows to `vavr` versions. I believe [0.9.1](https://github.com/vavr-io/vavr/issues?q=is%3Aopen+is%3Aissue+milestone%3Avavr-0.9.1) coming soon

**Pull request merged**